java/test/jmri/jmrit/logixng/configurexml/StoreAndLoadTest.java

```diff
@@ -2988,6 +2974,21 @@ private void addVariables(MaleSocket maleSocket) {
     }
 
 
+    private static final PrimitiveIterator.OfInt iterator =
+            new Random(215).ints('a', 'z'+10).iterator();
+
+    private String getRandomString(int count) {
+        StringBuilder s = new StringBuilder();
+        for (int i=0; i < count; i++) {
+            int r = iterator.nextInt();
+            if (i == 0 && r > 'z') r -= 10;     // The first char must be a character, not a digit.
+            char c = (char) (r > 'z' ? r-'z'+'0' : r);
+            s.append(c);
+        }
+        return s.toString();
+    }
```

Comment:
The purpose of this method is that before, whenether the LogixNG.xml file was created, there was a lot of changes to it since the female socket names are random and new random strings was created every time.

With this method, a known seed is always used, which results in the same random strings all the time. Therefore the changed LogixNG.xml file will have only some changes.